packages/wallets/src/features/cfd/screens/ChangePassword/InvestorPassword/MT5ChangeInvestorPasswordScreens.tsx

```diff
@@ -10,7 +12,25 @@ type TChangeInvestorPasswordScreenIndex = 'emailVerification' | 'introScreen' |
 const MT5ChangeInvestorPasswordScreens = () => {
     const [activeScreen, setActiveScreen] = useState<TChangeInvestorPasswordScreenIndex>('introScreen');
     const handleClick = (nextScreen: TChangeInvestorPasswordScreenIndex) => setActiveScreen(nextScreen);
-    const { hide } = useModal();
+    const { getModalState, hide } = useModal();
+    const { data } = useSettings();
+    const { mutate } = useVerifyEmail();
+    const { data: activeWallet } = useActiveWalletAccount();
+    const mt5AccountId = getModalState('accountId') || '';
+
+    const handleSendEmail = async () => {
+        if (data.email) {
+            await mutate({
+                type: 'trading_platform_investor_password_reset',
+                url_parameters: {
+                    redirect_to: platformPasswordResetRedirectLink('mt5', activeWallet?.is_virtual),
+                },
+                verify_email: data.email,
+            });
+
+            localStorage.setItem('trading_platform_investor_password_reset_account_id', mt5AccountId);
```

Comment:
**NOTE:** I need to add MT5 account ID to localStorage because we don't have it in reset email uri para, but need it to set a new investor password

Reply:
What if you pass `loginid` as payload, maybe the redirect link will come with loginid (just assumption)